deployments-review/apps/expense-service/src/main/resources/application.properties

```diff
@@ -0,0 +1,11 @@
+quarkus.swagger-ui.always-include=true
+
+#quarkus.datasource.devservices.image-name = docker.io/library/postgres:latest
+quarkus.hibernate-orm.sql-load-script=META-INF/import.sql
+quarkus.hibernate-orm.database.generation=create
+#quarkus.datasource.username = ${QUARKUS_DATABASE_USER}
```

Comment:
I assume that Quarkus has default env vars for datasources and that is why QUARKUS_DATASOURCE_USERNAME defined in the deployment resource is used.